Extension/src/LanguageServer/settings.ts

```diff
@@ -414,337 +517,157 @@ export class CppSettings extends Settings {
-    }
-
-    public get inlayHintsParameterNamesHideLeadingUnderscores(): boolean {
-        return super.Section.get<boolean>("inlayHints.parameterNames.hideLeadingUnderscores") === true;
-    }
-
-    public get inlayHintsReferenceOperator(): boolean {
-        return super.Section.get<boolean>("inlayHints.referenceOperator.enabled") === true;
-    }
-
-    public get inlayHintsReferenceOperatorShowSpace(): boolean {
-        return super.Section.get<boolean>("inlayHints.referenceOperator.showSpace") === true;
-    }
-
-    public get enhancedColorization(): boolean {
-        return super.Section.get<string>("enhancedColorization")?.toLowerCase() !== "disabled"
-            && this.intelliSenseEngine === "default"
-            && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast";
-    }
-
-    public get formattingEngine(): string | undefined {
-        return super.Section.get<string>("formatting")?.toLowerCase();
-    }
-
-    public get vcFormatIndentBraces(): boolean {
-        return super.Section.get<boolean>("vcFormat.indent.braces") === true;
-    }
-
+    public get defaultCompilerArgs(): string[] | undefined { return this.getAsArrayOfStrings("default.compilerArgs"); }
+    public get defaultCStandard(): string | undefined { return this.getAsString("default.cStandard"); }
+    public get defaultCppStandard(): string | undefined { return this.getAsString("default.cppStandard"); }
+    public get defaultConfigurationProvider(): string | undefined { return this.getAsString("default.configurationProvider"); }
+    public get defaultMergeConfigurations(): boolean | undefined { return this.getAsBoolean("default.mergeConfigurations"); }
+    public get defaultBrowsePath(): string[] | undefined { return this.getAsArrayOfStrings("default.browse.path") ?? undefined; }
+    public get defaultDatabaseFilename(): string | undefined { return this.getAsString("default.browse.databaseFilename"); }
+    public get defaultLimitSymbolsToIncludedHeaders(): boolean | undefined { return this.getAsBoolean("default.browse.limitSymbolsToIncludedHeaders"); }
+    public get defaultSystemIncludePath(): string[] | undefined { return this.getAsArrayOfStrings("default.systemIncludePath"); }
+    public get defaultEnableConfigurationSquiggles(): boolean | undefined { return this.getAsBoolean("default.enableConfigurationSquiggles"); }
+    public get defaultCustomConfigurationVariables(): { [key: string]: string } | undefined {return this.getAsKeyValueObject("default.customConfigurationVariables", "string", "string");}
+    public get useBacktickCommandSubstitution(): boolean | undefined { return this.getAsBoolean("debugger.useBacktickCommandSubstitution"); }
+    public get codeFolding(): boolean {return this.getAsString("codeFolding.enabled") === "enabled"; }
+    public get caseSensitiveFileSupport(): boolean { return !isWindows || this.getAsString("caseSensitiveFileSupport") === "enabled"; }
```

Comment:
`isCaseSensitiveFileSupportEnabled`

Reply:
Same issue as above, this involves making changes to package.json (presumably).

Reply:
I'm talking about the function name, not the setting name in package.json